labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -76,6 +83,6 @@ object CollectionExercise04 {
    * To keep it simple it's ok to use String.split to extract all words of a sentence.
    */
   def calcLengthLongestWord(lines: String*): Int = {
-    error("fix me")
+    lines.map(_.split(" ")).flatten.map(_.length()).max
```

Comment:
Did you know there's a function called flatMap? It does map and then flattens, and probably would make this line of code a tinsy bit nicer. For all I know, it's optimized to do it faster than calling map and flatten separately.